File: azure/repind-ri-gui-cs.old/src/app/shared/models/common/signatures/signature.ts
import { SignatureType } from './signature-type';

export class Signature {

    constructor(private _type: SignatureType, private _signature: string, private _site: string, private _date: Date) { }

    get type(): SignatureType {
        return this._type;
    }

    get signature(): string {
        return this._signature;
    }

    get site(): string {
        return this._site;
    }

    get date(): Date {
        return this._date;
    }

}

File: azure/repind-repind.old/msv/cf/graphql-bff-example/src/main/java/com/afklm/repind/msv/graphql/bff/example/model/Signature.java
package com.afklm.repind.msv.graphql.bff.example.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.*;
import org.springframework.format.annotation.DateTimeFormat;

import java.time.OffsetDateTime;
/**
 * Signature
 */
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class Signature {

    @JsonProperty("signatureType")
    private String signatureType;
    @JsonProperty("signatureSite")
    private String signatureSite;
    @JsonProperty("signature")
    private String signature;
    @JsonProperty("date")
    @DateTimeFormat(
            iso = DateTimeFormat.ISO.DATE_TIME
    )
    private OffsetDateTime date;
}
